waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TProcessorFactorySaslDecorator.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (C) 2016-2019 Expedia, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.server;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.shims.ShimLoader;
+import org.apache.hadoop.hive.thrift.HadoopThriftAuthBridge;
+import org.apache.thrift.TProcessor;
+import org.apache.thrift.TProcessorFactory;
+import org.apache.thrift.transport.TTransport;
+import org.apache.thrift.transport.TTransportException;
+
+
+public class TProcessorFactorySaslDecorator extends TProcessorFactory {
+
+    private  HadoopThriftAuthBridge.Server saslServer;
```

Comment:
We use indentations of 2 spaces in general. Do you mind changing this class to also have 2 spaces indentation?

Reply:
I've done this already, since it's easier from an IDE that has the settings for the spacing :)